xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -36,7 +36,7 @@ using namespace PVR;
 using namespace EPG;
 
 CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) :
-  CPVRChannelGroup(bRadio, bRadio ? PVR_INTERNAL_GROUP_ID_RADIO : PVR_INTERNAL_GROUP_ID_TV, g_localizeStrings.Get(19287)),
+  CPVRChannelGroup(bRadio, -1, g_localizeStrings.Get(19287)),
```

Comment:
Sure, but as long there are other places with identical convention (PVRChannelGroups.cpp:246) we should hit then all in a cosmetic cleanup PR.
I became a little bit careful during this thread - its particilar not clear to me what is cosmetic and what not. I reduced to minimum changes and I'm orienting on existing code (if possible)